docs/Running-Mastodon/Production-guide.md

```diff
@@ -1 +0,0 @@
-[The documentation has moved to its own repository](https://github.com/tootsuite/documentation/blob/master/Running-Mastodon/Production-guide.md)
```

Comment:
hm looks like I fucked up syncing my fork to master, is there something I can do to fix it?